satpy/FY4A_agri_introduction(EN).ipynb

```diff
@@ -0,0 +1,405 @@
+{
```

Comment:
<p>Radiances aren't available for this data, right ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pytroll/pytroll-examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Reflectance isn't available, but Radiance is available: </p><pre class="ql-syntax" spellcheck="false">  C07:
	    name: C07
	    wavelength: [3.5, 3.72, 4.0]
	    resolution:
	        2000: {file_type: agri_l1_2000m}
	        4000: {file_type: agri_l1_4000m}
	    calibration:
	      radiance:
	        standard_name: toa_outgoing_radiance_per_unit_wavenumber
	        units: "mW/ (m2 cm-1 sr)"
	      brightness_temperature:
	        standard_name: toa_brightness_temperature
	        units: "K"
	      counts:
	        standard_name: counts
	        units: "1"
</pre>

Reply:
<p>ok, good.</p><p> </p>